### DISCUSSION TITLE
Getting windows 11 version taskbar position to move to left or right just like in windows 10

### DISCUSSION THREAD

**Discussion by lizzail** [valinet/ExplorerPatcher#2914] [Ideas]
It's so bad microsoft scraps this feature on windows 11. I know you can do it when you change the taskbar style back to 10, but I would like to see this feature available for windows 11 version taskbar as well.

**Comment by pyrates999** [comment]
The code has to support it first.  Even if that code is disabled, it has to exist.  So unless it exists, it probably can't be implemented.
